integration-test/scripts/swagger-check.sh

```diff
@@ -41,6 +41,129 @@ Field_Separator=$IFS
 IFS=,
 set +e
 echo "Target branch for swagger check: ${CB_TARGET_BRANCH}"
+
+echo "Checking if there are any API changes between this and the target branch"
+git checkout $ghprbSourceBranch
+git reset --hard ${ghprbActualCommit}
+
+# list of the non-API modules
+declare -a nonApiModules=("cloud-reactor-api"
+  "common-handlebar"
+  "datalake"
+  "environment-log"
+  "idbmms-connector"
+  "redbeams-log"
+  "structuredevent-service-legacy"
+  "audit-connector"
+  "ccm-connector"
+  "cloud-reactor"
+  "docker-autoscale"
+  "environment"
+  "integration-test"
+  "redbeams-model"
+  "structuredevent-service"
+  "auth-connector"
+  "client-cm"
+  "cloud-template"
+  "common"
```

Comment:
Looks almost good, unfortunately, common is included in environment-api, so if you change things there it might be still unnoticed

https://github.com/hortonworks/cloudbreak/blob/7959221b2102b2ac5c4ff5899dcdb687d8a968b9/environment-api/build.gradle#L17

Can we also somehow check the transitive dependencies of the modules? Maybe it does not worth it, if it is too much effort...

Reply:
I cannot do the transitive dependency check :( Or it would take 1 to 2 weeks.

Reply:
Removed the common from the non-API modules

Reply:
Understood, probably does not worth the effort...